src/main/java/graphql/annotations/GraphQLAnnotations.java

```diff
@@ -618,11 +628,13 @@ protected GraphQLFieldDefinition getField(Method method) throws GraphQLAnnotatio
             outputTypeFunction = typeFunction;
         }
 
-        GraphQLOutputType type = (GraphQLOutputType) outputTypeFunction.buildType(method.getReturnType(), annotatedReturnType);
-        GraphQLOutputType outputType = method.getAnnotation(NotNull.class) == null ? type : new GraphQLNonNull(type);
+        GraphQLOutputType outputType = (GraphQLOutputType) outputTypeFunction.buildType(method.getReturnType(), annotatedReturnType);
+        outputType = method.getAnnotation(NotNull.class) == null ? outputType : new GraphQLNonNull(outputType);
```

Comment:
1. Why NotNull and not GraphQLNotNull?
2. Please use isAnnotationsPresent

Reply:
I did not commit it yet :-)

Reply:
ohh, sorry. I thought you were because there are not conflicts now